out/bootstrap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: "cluster.k8s.io/v1alpha1"
```

Comment:
This file should not be part of the commit. probably add the out folder to the .gitignore

Reply:
+1 for adding to .gitignore